core/modules/css/CssAccess.cc

```diff
@@ -626,6 +602,13 @@ CssAccess::createTable(std::string const& dbName,
         _storePacked(tableKey + "/partitioning", partMap);
     }
 
+    // save shared scan info
+    std::map<std::string, std::string> partMap{
+        std::make_pair("lockInMem", std::to_string(int(scanParams.lockInMem))),
+        std::make_pair("scanSpeed", std::to_string(int(scanParams.scanSpeed)))
+    };
+    _storePacked(tableKey + "/sharedScan", partMap);
+
```

Comment:
This creates `sharedScan` key even for non-partitioned tables which does not make much sense, I propose to move this inside `if` above.